content/en/real_user_monitoring/faq/session_replay_service_worker.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Allow Third-Party Service Worker for Session Replay
+kind: faq
+further_reading:
+    - link: '/real_user_monitoring/session_replay/'
+      tag: 'Get Started'
+      text: 'Session Replay'
+---
+
+## Third-Party Cookie Blocking And Session Replay
+
+Session Replay uses a *Service Worker* in a different domain (*session-replay-datadoghq.com*) to provide the best experience possible whilst protecting your privacy and guaranteeing the safety of your data.
+
+If you have blocked third-party cookies in your browser settings, or if your browser blocks them by default, this can prevent the service worker from registering correctly.
+
+### Allow an exception
+
+Datadog's recommended solution is to make an exception to your third-party cookie blocking, to allow Session Replay's service worker to work and enjoy the most performant replaying experience:
+
+{{< img src="real_user_monitoring/session_replay/allow-3p-serviceworker-1.png" alt="Allow Session Replay Third-Party Service Worker" >}}
```

Comment:
Moving this image to the section above. 

```suggestion
```

Reply:
The two screenshots are steps to make the cookie exception, I'm not sure it makes sense to place them separately. Should I run this through Design to see if there's a chance to improve the artwork?

Reply:
Yes I agree with Alfonso, the screenshots are closely related steps to reactivate 3rd party cookies for DD and should not be separated; to make it clearer than they are in fact steps the user can/should follow, i would suggest to add a small line / paragraph indicating that these screenshots are part of the process to follow.

Reply:
Ahh, that sounds good! I didn't know they were in order. Let's add some instructions in then 😛